ADOPTERS.md

```diff
@@ -92,6 +92,19 @@ Details (optional):
 - A huge fleet of OpenShift and Kubernetes clusters, each using Prometheus Operator
 - All managed by [Project Syn](https://syn.tools/), leveraging Commodore Components like [component-rancher-monitoring](https://github.com/projectsyn/component-rancher-monitoring) which re-uses Prometheus Operator
 
+## Innovaccer ##
```

Comment:
We like this file to be alphabetically sorted so no company is last for any specific reason, do you mind doing that, thanks!